lingpy-server.sh

```diff
@@ -1,2 +1,2 @@
 cd app
-python server.py start
+python3 server.py start
```

Comment:
that depends on your working environment, mine is using python as basic code for python3, as are most archlinux systems,

Reply:
I think `python3` is supposed to be the canonical executable name, at least until January when Python2 will be deprecated.  But yes, minor issue.